File: fixtures/node_output/get_basic_auth.js
var request = require('request');

var options = {
    url: 'https://api.test.com/',
    auth: {
        'user': 'some_username',
        'pass': 'some_password'
    }
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);

File: fixtures/elixir_output/get_basic_auth.ex
method = :get
url = "https://api.test.com/"
headers = []
body = nil
options = [hackney: [basic_auth: {~s|some_username|, ~s|some_password|}]]
params = []

request = %HTTPoison.Request{
  method: method,
  url: url,
  body: body,
  headers: headers,
  options: options,
  params: params,
}

response = HTTPoison.request(request)
